src/notfication/notify.py

```diff
@@ -0,0 +1,28 @@
+from email.message import EmailMessage
+import smtplib
+import os
+import ssl
+# from dotenv import load_dotenv
+# load_dotenv()
+
+class Notify:
+    # __serviceAccEmail = os.getenv('SERVICE_EMAIL')
+    # __serviceAccPassword = os.getenv('SERVICE_PASSWORD')
+
+    _subject = " Something went wrong !!"
+
+    def email(errorMessage:str,to:list[str]):
+
+        for dest in to:
+            em = EmailMessage()
+            em['From']=Notify.__serviceAccEmail
+            em['Subject']= Notify._subject
+            em.set_content(errorMessage)
+            em['To']= dest
+            context = ssl.create_default_context()
+            with smtplib.SMTP_SSL('smtp.gmail.com', 465 , context=context) as s:    
+               
+                s.login(Notify.__serviceAccEmail, Notify.__serviceAccPassword)
```

Comment:
Why are we performing the login multiple times in the for loop?\
The login event will be triggering before sending the email to each individual which is unnecessary.\
Can we move this to outside the for loop as we just need to login once.

Reply:
done